### DISCUSSION TITLE
The custom RPC does not appear on other Discord accounts

### DISCUSSION THREAD

**Discussion by tokkegamer622** [maximmax42/Discord-CustomRP#240] [Q&A]
hello, I have used Custom rpc in discord but it only appears on my account, but does not appear on other people's accounts. can anyone help me? I have used VPN / proxy but it still only appears in my account. so i can't prank other people
![Screenshot 2024-04-25 202406](https://github.com/maximmax42/Discord-CustomRP/assets/168094165/c6695ff5-917d-431f-b47d-01dabbcb8d4f)
![Screenshot 2024-04-25 202440](https://github.com/maximmax42/Discord-CustomRP/assets/168094165/c708ce5e-44f7-4659-9d9e-78c78914cdab)

**Comment by maximmax42** [comment]
Is the app stuck on "Updating presence..."? If yes, check the image link, something might be wrong with it.
Otherwise:
![](https://github.com/maximmax42/CustomRP-Docs/assets/2225711/3a8f3040-0dfa-4883-b4f2-4142474553f6)

**Reply by tokkegamer622** [reply]
i am already turn it on, but when using another account, it only appears for about 1-2 seconds and then disappears, or disappears when refreshed the discord on chrome

**Reply by maximmax42** [reply]
Weird. Are you sure the app doesn't get stuck on "Updating presence" for a long time?

**Reply by tokkegamer622** [reply]
yes, but i have a feeling that the problem is on my connection, im using a router that many blocked website, i cant send files to discord, so i need to use vpn

**Reply by maximmax42** [reply]
Connection between CustomRP and your Discord client is local, any blocks shouldn't affect anything. But I dunno.

**Reply by tokkegamer622** [reply]
this problem is happen too when i am using custom rpc on vencord

**Reply by maximmax42** [reply]
Hm. I'm not sure what to do in that case.
